src/features/dataset/modal/EditDatasetTitleModal.tsx

```diff
@@ -15,8 +15,8 @@ export interface EditDatasetTitleModalProps {
 
 const EditDatasetTitleModal: React.FC<EditDatasetTitleModalProps> = ({ title, onCommit }) => {
   const dispatch = useDispatch()
-  const titleEditError = useSelector(selectTitleError)
-  const loading = useSelector(selectIsDatasetLoading)
+  const titleEditError = ''
+  const loading = ''
```

Comment:
I don't understand this change.

Reply:
These were changed as the actions they use are no longer present and it was causing compiling errors.  I just removed the component as we are no longer using it.